.github/actions/README.md

```
@@ -66,6 +66,27 @@ This action runs `tools/setup-tools --skip-go --skip-docker` in auto mode, which
 - Skips Go (handled by `actions/setup-go`) and Docker (pre-installed on runners)
 - Uses the same installation logic as local development
 
+#### `install-go-licenses/`
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)